tests/unit/drivers/test_cache_driver.py

```diff
@@ -76,8 +76,8 @@ def test_cache_driver_from_file(tmpdir, test_metas):
     folder = os.path.join(test_metas["workspace"])
     bin_full_path = os.path.join(folder, filename)
     docs = list(random_docs(10, embedding=False))
-    pickle.dump([doc.id for doc in docs], open(f'{bin_full_path}.ids', 'wb'))
-    pickle.dump([doc.content_hash for doc in docs], open(f'{bin_full_path}.cache', 'wb'))
+    pickle.dump({doc.id: i for i, doc in enumerate(docs)}, open(f'{bin_full_path}.ids', 'wb'))
+    pickle.dump({doc.content_hash: i for i, doc in enumerate(docs)}, open(f'{bin_full_path}.cache', 'wb'))
```

Comment:
Should this be `i`? I think it should be the field by which we cache, right?

Reply:
Cause here you have

```
self.query_handler.id_to_cache_val[doc_id] = data
```

Reply:
Nice catch! 
That was confusing. I changed it.
The test was passing because it just checks if it can load the files and not if the values make sense.